pallets/ajuna-awesome-avatars/src/types.rs

```diff
@@ -60,7 +60,7 @@ pub struct SeasonMetadata {
 }
 
 #[derive(Encode, Decode, Clone, Default, TypeInfo, MaxEncodedLen)]
-pub struct AwesomeAvatar {
+pub struct Avatar {
```

Comment:
Changed from `AwesomeAvatar` -> `Avatar` because "awesome" is implied :)

Reply:
I agree :D changed!